app/controllers/sites_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SitesController < ApplicationController
+  before_action :set_site
+  load_and_authorize_resource
```

Comment:
The load will never be run, because set_site will first assign `@site`, is that intended?